main/py-oauth2client/APKBUILD

```diff
@@ -1,40 +1,49 @@
 # Maintainer: Peter Bui <pnutzh4x0r@gmail.com>
 pkgname=py-oauth2client
 _pkgname=oauth2client
-pkgver=1.4.12
+pkgver=4.0.0
 pkgrel=0
 pkgdesc="A client library for OAuth 2.0"
 url="https://github.com/google/oauth2client"
 arch="noarch"
-license="Apache"
-depends="python2 py-asn1 py-httplib2 py-asn1-modules py-rsa py-six"
-depends_dev=""
-makedepends="python2-dev py-setuptools"
-install=""
-subpackages=""
+license="ASL 2.0"
+depends="py-asn1 py-httplib2 py-asn1-modules py-rsa py-six"
```

Comment:
I know...there are 5 or 6 more packages that have somehow a relationship with `gcalcli`. Most of them should be in `community` anyway from my point of view